run_tests

```diff
@@ -55,7 +55,6 @@ EXCLUDE["X64"]=[
 ]
 
 EXCLUDE["RISCV64"]=[
-    "Access",
```

Comment:
The test finally run correctly and are picking up that we do still need to exclude the session `InfoFlow` (and its dependencies) for RISC-V.